README.md

```diff
@@ -163,7 +171,7 @@ async fn main() -> Result<(), sqlx::Error> {
     // Create a connection pool
     let pool = PgPool::builder()
         .max_size(5) // maximum number of connections in the pool
-        .build(env::var("DATABASE_URL")?).await?;
+        .build(&*env::var("DATABASE_URL").unwrap()).await?;
```

Comment:
It looks like this missing a `&` so good catch but we definitely want the `?` over the `.unwrap()`